### DISCUSSION TITLE
Redacting/hiding an entire collection

### DISCUSSION THREAD

**Discussion by umn-annakfowler** [Symbiota/community-board#496] [Q&A] [answered]
Hello, I'm the data manager for the Minnesota Biodiversity Atlas, and I'm looking into making a test collection to use for running new SQL scripts in the back-end, testing out new procedures that would change lots of data, etc. 

What I don't see in the Symbiota docs, support group videos, or the metadata settings within the front-end route of creating a new collection is a way to make the **entire collection** private except to administrators of the collection. Currently it is visible even when not logged in to the portal.

Does this exist as a front-end feature? Is there an SQL query that would do it?

**Comment by themerekat** [comment, marked as answer]
Hi @annakfowler, good question! There is no way to make an entire collection private. Symbiota's ethos is public data sharing, so systems of keeping data private have not been extensively developed, except the ability to hide locality data for taxa.
